Content.Server/Objectives/Conditions/StealCondition.cs

```diff
@@ -14,34 +14,30 @@ public sealed class StealCondition : IObjectiveCondition, ISerializationHooks
     {
         private Mind.Mind? _mind;
         [DataField("prototype")] private string _prototypeId = string.Empty;
-        [DataField("amount")] private int _amount = 1;
```

Comment:
i'd rather you adjust the loc logic to use a different string if amount is >1 instead of removing the option outright

Reply:
It's not removing functionality because the amount thing does not function. If you search in Discord it's been brought up a few times before. That's also why despite a bunch of steal objectives being added, none of them require you steal more than 1. Any time someone has tried to add one for say, gold bars or something, it did not work. So, personally, I think that's a bit out of scope.